src/Stubs/common/DenormalizerInterface.stubphp

```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace Symfony\Component\Serializer\Normalizer;
+
+interface DenormalizerInterface
+{
+    /**
+     * @template     T
+     * @psalm-param  class-string<T> $type
```

Comment:
`@param mixed $data` is required for strict mode (before this stub Psalm used the original phpdoc)